scripts/toolboxSrc/setIsEnterprise.ts

```diff
@@ -0,0 +1,47 @@
+import getRethink from '../../packages/server/database/rethinkDriver'
+import getPg from '../../packages/server/postgres/getPg'
+
+export default async function setIsEnterprise() {
+  const r = await getRethink()
+
+  console.log('Updating tier to "enterprise" for Organization and OrganizationUser tables in RethinkDB')
```

Comment:
+1 before going any further, we should check if `process.env.IS_ENTERPRISE === 'true'`. that way we're sure they've done everything they need to do to achieve success.

Reply:
I added this check @mattkrick